_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Organize projects and monitor</td>
+    <td>Organize ESDC projects in GCcode and monitor usage</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Enable connectivity to the Internet</td>
+    <td>Enable projects in GCcode to deploy to public cloud service providers</td>
+    <td><b>SSC</b></td>
+    <td>
+      Cloud CoE<br>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2"><b>Development</b></td>
+    <td>GCcode:<br>
+    Enable DevOps capabilities</td>
+    <td>
+    Ensure DevOps capabilities are available in GCcode, such as:
+    <ul>
+      <li>Continuous Integration (CI)</li>
+      <li>Continuous Deployment (CD)</li>
+    </ul>
+    </td>
+    <td><b>SSC</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>
+  <tr>
+    <td>Migrate projects to GCcode</td>
+    <td>Migrate software-related projects, scripts and system/database configurations to the chosen VCS, and gradually sunset legacy version control systems
+    </td>
+    <td><b>Development teams</b></td>
+    <td>
+    DevCoP<br>
+    Operations teams<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="1"><b>Communications</b></td>
+    <td>Train staff</td>
+    <td>Offer training resources to IITB staff related to DevOps and VCS</td>
+    <td><b>Learning Branch</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>  
+</table>
+
+## Measuring the Strategy's success
+
+This Strategy's success will be measured by comparing the following metrics against the conventional IT project methodology.
+
+The metrics are defined as follows:
+
+1. *Number of ESDC projects (open vs closed) and users in each VCS*
+1. *Level of activity of ESDC projects (e.g., issues, commits, integrations) in each VCS*
+1. *User Satisfaction*
+1. *?*
+
+<table>
+<tr>
+    <th rowspan="2">Metric</th>
+    <th colspan="2">Collection Method</th>
+</tr>
+<tr>
+    <th>Target Model</th>
+    <th>Conventional Model</th>
+</tr>
+<tr>
+    <td>Number of ESDC projects (open vs closed) and users in each VCS</td>
+    <td>
+    <i>Automatic</i><br>
+    Automatically fetch metrics from platforms
+    </td>
+    <td>
+    <i>Manual</i><br>
+    Manually copy metrics from platforms
+    </td>
+</tr>
+<tr>
+    <td>Level of activity of ESDC projects (e.g., issues, commits, integrations) in each VCS</td>
+    <td>
+    <i>Automatic</i><br>
+    Automatically fetch metrics from platforms
+    </td>
+    <td>
+    <i>Manual</i><br>
+    Manually copy metrics from platforms
+    </td>
+</tr>
+<tr>
+    <td>User Satisfaction</td>
+    <td>
+    <i>Manual</i><br>
+    Client survey
+    </td>
+    <td>
+    <i>Manual</i><br>
+    Client survey
+    </td>
+</tr>
+<tr>
+    <td>?</td>
+    <td>
+    TBC
+    </td>
+    <td>
+    TBC
+    </td>
+</tr>
+</table>
+
+*Manual*: the collection of data requires manual intervention (e.g., surveys, interviews, emails, spreadsheet updates).
```

Comment:
Regarding metrics, I've been asked to use quantatitive indicators: so increase is not enough, they were asking for how much? If we're currently 20% on Git and rest is distributed in many other, we would like to be at 80% in 2 years.

Reply:
Agreed. This goes into roadmap territories. The Metrics section can simply list them while the Roadmap section would add objectives. I would keep the roadmap for version 0.02